ext/ctype/ctype.stub.php

```diff
@@ -2,24 +2,57 @@
 
 /** @generate-class-entries */
 
+/**
+ * @compile-time-eval
+ */
 function ctype_alnum(mixed $text): bool {}
 
+/**
+ * @compile-time-eval
+ */
 function ctype_alpha(mixed $text): bool {}
 
+/**
+ * @compile-time-eval
+ */
 function ctype_cntrl(mixed $text): bool {}
 
+/**
+ * @compile-time-eval
+ */
 function ctype_digit(mixed $text): bool {}
 
+/**
+ * @compile-time-eval
```

Comment:
is `ctype_lower` still somehow locale dependent?

test failure:
```
 009+ ctype_lower(\xe4): n
009- ctype_lower(\xe4): y
     preg_match(\w, \xe4): int(1)
========DONE========
FAIL Do not inherit LC_CTYPE from environment [ext/ctype/tests/lc_ctype_inheritance.phpt] 
```

Reply:
Yes, the ctype functions were not part of https://wiki.php.net/rfc/strtolower-ascii.